packages/jest-docblock/src/__tests__/index.test.ts

```diff
@@ -170,16 +152,11 @@ describe('docblock', () => {
 
   it('parses directives out of a docblock with line comments', () => {
     const code =
-      '/**' +
-      EOL +
-      '' +
-      ' * @team foo' +
-      EOL +
-      '' +
-      ' * // TODO: test' +
-      EOL +
-      '' +
-      ' */';
+      String(
+        String(String('/**' + EOL) + ' * @team foo' + EOL) +
```

Comment:
same here, from readability perspective this is not good
try to wrap just `EOL` instances like `String(EOL)`